src/locking/LockingManager.ts

```diff
@@ -171,10 +127,64 @@ export default class LockingManager {
       hostname: Utils.getHostname()
     };
     // Set expiration date
-    if (expirationTimeSeconds > 0) {
-      lock.expirationDate = moment(lock.timestamp).add(expirationTimeSeconds, 'seconds').toDate();
+    if (lockValiditySecs > 0) {
+      lock.expirationDate = new Date(lock.timestamp.getTime() + (lockValiditySecs * 1000));
```

Comment:
If you rely on unix timestamp everywhere, you will simplify and speed all expire test and avoid to manipulate `Date` object.

Reply:
I'm ok with the date, it's more readable in the DB